presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCacheConfig.java

```diff
@@ -47,7 +49,10 @@ public void testDefaults()
                 .setTimeoutThreads(64)
                 .setCacheQuotaEnabled(false)
                 .setShadowCacheEnabled(false)
-                .setShadowCacheWindow(new Duration(7, DAYS)));
+                .setShadowCacheWindow(new Duration(7, DAYS))
+                .setTtlEnabled(false)
+                .setTtlCheckInterval(new Duration(1, HOURS))
```

Comment:
better to keep them alphabetical

```suggestion
                .setTtlCheckInterval(new Duration(1, HOURS))
                .setTtlEnabled(false)
```